### ISSUE TITLE
Missing timeout for requests.Session object?

### ISSUE THREAD

**Issue by fossilet** [pylint-dev/pylint#9517]
### Question

missing-timeout / W3101 reports in my codebase `requests.get` calls, but not for:

```python
session = requests.Session()
session.get(url)
```
Is it possible to make pylint also warn about these cases?

### Documentation for future user

in code and docs.

**Comment by joel6948** [comment]
https://github.com/pylint-dev/pylint/pull/9525#issue-2217965137

i have added which may help in solving it 
please let me know issues regarding it 

use the command below for checks
pylint --load-plugins=pylint_plugins.requests_timeout_plugin your_code.py

**Comment by joel6948** [comment]
https://github.com/pylint-dev/pylint/pull/9526#issue-2218001689 bit issue fixes in it
